service/controller/key/key.go

```diff
@@ -362,16 +360,6 @@ func MasterCount(customObject v1alpha1.AWSConfig) int {
 	return len(customObject.Spec.AWS.Masters)
 }
 
-func MasterImageID(customObject v1alpha1.AWSConfig) string {
```

Comment:
We don't want to use the `ImageID` from the CR spec anymore. Instead it comes from the release's OS version combined with the region.